src/content/nav/en.ts

```diff
@@ -19,6 +19,7 @@ export default {
 	'reference.other': 'Other development APIs',
 	'reference.syntax': 'Astro Template Syntax',
 	'reference.experimental': 'Experimental features',
+	'reference.ai-resources': 'AI resources',
```

Comment:
I thought it was probably best to create a separate section. I was inspired by the Cloudflare docs that have a similar section in the sidebar